docs/resources/redis.md

```diff
@@ -54,6 +54,7 @@ The following arguments are supported:
 * `mode` - (Required) Determines the configuration of Cloud DB for Redis. When the CLUSTER setting is used, the `is_ha` setting is ignored. Options: `CLUSTER`, `SIMPLE`.
 * `image_product_code` - (Optional) Image product code to determine the Redis instance server image specification to create. If not entered, the instance is created for default value. It can be obtained through [`data.ncloud_redis_image_products`](../data-sources/redis_image_products.md).
 * `product_code` - (Optional) Sets the server specifications of the Cloud DB for Redis instance to be created. It can be obtained through [`data.ncloud_redis_products`](../data-sources/redis_products.md). Default : Minimum specifications(1 memory, 2 cpu)
+* `engine_version_code` - (Optional) Redis engine version code. Only entered when `generation_code` is G3, If not entered, generate with the latest version currently available.
```

Comment:
Redis engine version code. If not entered, generate with the default version currently available.